ecs-cli/modules/cli/local/network/network.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2015-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Nit: I wonder if the name of the file should be something different, since both network and container creation happen here.

Reply:
I tried splitting it up into `network.go`, `container.go` and `setup.go` but ended up not liking it as much because all these functions were scattered around and we lost locality. I'm down to just rename this file though or package 👍. Let me know if you can have any suggestions for a better name.

Reply:
On second thought, I renamed the file `setup.go` since the only public function in the file is `Setup()`. The next task is to implement `teardown.go` with the `Teardown()` function. What do you think?

Reply:
Sounds good 👍